packages/cosmos_ui_components/lib/components/template/cosmos_wallets_list_view.dart

```diff
@@ -6,27 +7,38 @@ class CosmosWalletsListView extends StatelessWidget {
   final List<WalletInfo> list;
   final WalletInfo selectedWallet;
   final void Function(int) onClicked;
+  final void Function(WalletInfo) onEditIconPressed;
+  final bool isEditing;
 
   const CosmosWalletsListView({
     Key? key,
     required this.list,
     required this.selectedWallet,
     required this.onClicked,
+    this.isEditing = false,
+    required this.onEditIconPressed,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: list
-          .mapIndexed(
-            (index, wallet) => CosmosWalletListItem(
-              name: wallet.name,
-              address: wallet.address,
-              isSelected: selectedWallet.address == wallet.address,
-              onClicked: () => onClicked(index),
-            ),
-          )
-          .toList(),
+    return SingleChildScrollView(
+      child: Column(
+        children: list
+            .mapIndexed(
+              (index, wallet) => CosmosWalletListItem(
+                name: wallet.name,
+                address: wallet.address,
+                isSelected: selectedWallet.address == wallet.address,
+                onClicked: () => onClicked(index),
+                suffix: isEditing
+                    ? CosmosImageButton(assetUrl: 'assets/images/more.png', onPressed: () => onEditIconPressed(wallet))
+                    : selectedWallet.address == wallet.address
+                        ? CosmosImageButton(assetUrl: 'assets/images/check.png', onPressed: () {})
```

Comment:
this is not a button but rather an indicator, so you should either use plain `Image.asset` or at least pass `null` to `onPressed`, otherwise there will be splash effect visible when touching